telco-examples/edge-clusters/aarch64/README.md

```diff
@@ -0,0 +1,114 @@
+# Edge clusters for Telco using aarch64 architecture
+
```

Comment:
We should perhaps put something here about this being a tech-preview scenario?

Reply:
yes I will change it now and I will update the metal3 helm chart info after merging the PR with the changes. Let me update it and I will request a review changing the state to ready for review! 
Thanks Steven